### DISCUSSION TITLE
Multicolor printing and color sequence for first layer

### DISCUSSION THREAD

**Discussion by ClamsyHub** [OrcaSlicer/OrcaSlicer#1036] [Q&A]
Hello,
I have to do with a very filligree print. Therefore I am already printing with a 0.2 Nozzle.

Now I have a white background with a green motif in the first layer. The green motif has very thin lines, which he also prints with 0.2 Nozzle super.  But the printer starts with the white color and then prints green. Consequently, this very thin line becomes invisible, because it is hidden in the connection to the two surrounding white lines. If it printed green first and then white, this would be bypassed and the fine green lines would be visible.

Therefore, how can I specify in the slicer which color order it should use?

I have: Slot 1 - black, Slot 2 - green, Slot 3 - red, Slot 4 - white.

The Slicer print the following order:
1. white
2. green
3. red
4. black

And I would like to print as follows:
1. green
5. black
6. red
7. white

Please help.

**Comment by ClamsyHub** [comment]
It is clear that the order may well change even during the further layers - I have just noticed. This also makes sense to save filament (for the cleaning processes).

But in my case I don't care if I have 20 times more cleaning. In my case, it is important to me that these fine colors remain visible, and for me, each layer may also have to adhere to the exact order I have specified.

So I have more cleaning processes by forcing the sequence, but a clean result of the color print.

**Comment by ClamsyHub** [comment]
Maybe these photos will help to better understand the problem.

https://postimg.cc/gallery/w7WW6gC

**Comment by ClamsyHub** [comment]
I have found at least a "temporary solution".
It goes about the flush volumes.
You have to adjust the flush volumes between the colors (a difference of 1 is enough). You have to play a bit until you get it, but this way you can at least control the colors a bit, so that you get to the desired order.

Not a nice solution, but at least it's a solution.

**Comment by qhga** [comment]
For anyone searching for a solution. It is already implemented in Orca/Bambu Slicer, it is just "hard" to find.

1. Click on the bolt / nozzle symbol next to the 3D print plate (Customize current plate)
2. Adjust "First layer filament sequence" to `Customize`
3. Pick desired order

**Reply by boehser-enkel** [reply]
> For anyone searching for a solution. It is already implemented in Orca/Bambu Slicer, it is just "hard" to find.
> 
> 1. Click on the bolt / nozzle symbol next to the 3D print plate (Customize current plate)
> 2. Adjust "First layer filament sequence" to `Customize`
> 3. Pick desired order

But it's only available for the first layer, not the others? In Bambu there is also an option for the rest

**Reply by dabalroman** [reply]
![image](https://github.com/user-attachments/assets/a6767247-4acf-48d2-bba3-3e42ec07b1b3)

You can do as many sequences as you want to.

**Reply by rodgrossi** [reply]
you saved me, thanks
